src/views/Home/components/Prizes/prizes.scss

```diff
@@ -1,3 +1,70 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
 
-/* Delete these lines when you begin development */
+.Prizes {
+  margin: 0;
+  background: url("~images/components/prizes/background.svg") repeat-y;
```

Comment:
Again, I'd advise against confining the background to a single CSS prop; it looks like the image is being cropped. A CSS class would lend enough granularity to scale the entire image! The background layers have a bit more complexity than a single graph; I think a separate background container will really help here.

implementation:
![Screenshot from 2020-12-09 09-22-49](https://user-images.githubusercontent.com/19658882/101664309-8cb4c400-3a00-11eb-9678-1a57ceb7c93b.png)

design:
![Screenshot from 2020-12-09 09-23-13](https://user-images.githubusercontent.com/19658882/101664326-90e0e180-3a00-11eb-81eb-114db3b1e0a2.png)

Reply:
Also we can include the glowing shadow effect using CSS! https://www.w3schools.com/css/css3_shadows.asp

Reply:
@mocklee A separate background container would make sense so I will update #91 and this PR. However, I'm still unsure how to repeat the SVG without using the `background-repeat` prop. This was needed so the background can cover the entire section on small screens.

Reply:
instead of repeating the grid SVG, its height & width can be set proportional to a container. if it doesn't size properly on smaller screens, consider jumping into the SVG & defining `preserveAspectRatio` to lend a bit more flexibility with resizing: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio